index.html

```diff
@@ -294,116 +276,123 @@ <h4>Multi-Post Stories</h4>
-          me.
-        </p>
-      </section>
+          <p class="about">
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hestiate to
+            contact me.
+          </p>
 
-      <section class="social-media-container">
-        <h4 class="social-media-title">Let's Connect</h4>
-        <ul class="social-media-content">
-          <li>
-            <a href="#">
-              <img
-                class="social-media-icons"
-                src="./images/icons/twitter.png"
-                alt="Twitter Icon"
-              />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img
-                class="social-media-icons"
-                src="./images/icons/linkedin.png"
-                alt="LinkedIn Icon"
-              />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img
-                class="social-media-icons"
-                src="./images/icons/media.png"
-                alt="Media Icon"
-              />
-            </a>
-          </li>
-          <li>
-            <a href="#">
+          <div class="social-media-container-bottom">
+            <h4 class="social-media-title">Let's Connect</h4>
+            <ul class="social-media-content">
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="./images/icons/twitter.png"
+                    alt="Twitter Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="./images/icons/linkedin.png"
+                    alt="LinkedIn Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="./images/icons/media.png"
+                    alt="Media Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="./images/icons/github.png"
+                    alt="Github Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="./images/icons/victory.png"
+                    alt="Victory Icon"
+                  />
+                </a>
+              </li>
+            </ul>
+            <div class="btn-container">
+              <button type="submit" class="view-btn">Get my resume</button>
```

Comment:
Great work @fullstop125 🤝

- [ ] You may please try to align this button correctly. Try to push your button to the right a bit. Take another look at the Figma template.
- Your design
![button align](https://user-images.githubusercontent.com/63917725/178039693-836b2bfa-27ca-4129-bf11-e4191eca92aa.PNG)

Reply:
resolved